client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -34,29 +36,65 @@ export function hasStripeValidCredentials( method ) {
 class PaymentMethodStripe extends Component {
 
 	static propTypes = {
+		stripeConnectAccount: PropTypes.shape( {
+			connectedUserID: PropTypes.string,
+			displayName: PropTypes.string,
+			email: PropTypes.string,
+			firstName: PropTypes.string,
+			isActivated: PropTypes.bool,
+			lastName: PropTypes.string,
+			logo: PropTypes.string,
+		} ),
 		method: PropTypes.shape( {
 			settings: PropTypes.shape( {
-				title: PropTypes.shape( {
-					id: PropTypes.string.isRequired,
-					label: PropTypes.string.isRequired,
-					type: PropTypes.string.isRequired,
-					value: PropTypes.string.isRequired,
-				} ),
+				apple_pay: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				capture: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				secret_key: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				publishable_key: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				testmode: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				test_publishable_key: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
+				test_secret_key: PropTypes.shape( { value: PropTypes.string.isRequired } ).isRequired,
 			} ),
 		} ),
-		translate: PropTypes.func.isRequired,
```

Comment:
Was it intentional to remove this? Looks like it's still used.

Reply:
I have not tended to proptype our translate mixin - this was in there from previous work. I could put it back if you feel strongly about it

Reply:
I don't feel strongly about it. I'm fine with omitting.

Reply:
Ha, I'll stop mentioning this in reviews then ;)